docs/src/main/asciidoc/update-quarkus.adoc

```diff
@@ -38,13 +36,13 @@ include::{includes}/prerequisites.adoc[]
 
 . Create a working branch for your project by using your version control system.
 
-. Install the *latest version* of the Quarkus CLI to use in the next step. For more information, see link:https://quarkus.io/guides/cli-tooling#installing-the-cli[Installing the CLI].
-Confirm the version number by using `quarkus -v`.
+. To use the Quarkus CLI in the next step, link:https://quarkus.io/guides/cli-tooling#installing-the-cli[install the latest version of the Quarkus CLI].
+Confirm the version number using `quarkus -v`.
 
 . Go to the project directory and update the project to the latest stream:
 +
 [role="primary asciidoc-tabs-sync-cli"]
-.Using Quarkus CLI
+* Using Quarkus CLI
```

Comment:
We need to discuss this change.

Reply:
I've reverted this particular change for now. 
If needed, I'll make any related updates in a separate future PR after the team sorts out this particular detail.